features/app_and_device_attributes.feature

```diff
@@ -6,19 +6,22 @@ Feature: App and Device attributes present
   Scenario: App and Device info is as expected
     When I run "AppAndDeviceAttributesScenario"
     And I wait to receive an error
-    Then the error is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    Then the error is valid for the error reporting API
     And the error "Bugsnag-API-Key" header equals "12312312312312312312312312312312"
 
     # Device
 
-    And the error payload field "events.0.device.osName" equals "iOS"
+    And the error payload field "events.0.device.osName" equals the platform-dependent string:
+      | ios   | iOS    |
+      | macos | Mac OS |
     And the error payload field "events.0.device.jailbroken" is false
     And the error payload field "events.0.device.osVersion" matches the regex "\d+\.\d+"
     And the error payload field "events.0.device.manufacturer" equals "Apple"
     And the error payload field "events.0.device.locale" is not null
     And the error payload field "events.0.device.id" is not null
     And the error payload field "events.0.device.model" matches the test device model
-    And the error payload field "events.0.device.modelNumber" is not null
+    # modelNumber is not available on macOS
```

Comment:
Do we have a ticket to cover things like this, or do we need to put an iOS specific step in before we can merge this?

Reply:
This will be resolved by adding a nullable tag to the platform-dependent test assertion steps.